chirp/drivers/radtel_rt490.py

```diff
@@ -421,6 +421,9 @@ class RT490Radio(chirp_common.CloneModeRadio):
                (0x3FC0, 0x4000)
               ]
 
+    _valid_chars = chirp_common.CHARSET_ALPHANUMERIC + \
+        "`~!@#$%^&*()-=_+[]\\{}|;':\",./<>?"
+
```

Comment:
This isn't used anywhere so this won't have any effect. I assume you want to set `valid_characters` in `get_features()` ?

Reply:
And yet it works, so I don't know what to tell you.

I took the example directly from mml_jc8810.py

Reply:
It really doesn't. I changed it to just `'a'` and I can still put in names like "FOO". With and without your patch, I can only put in uppercase characters.

The `mmp_jc8810.py` actually references it exactly where I said, which is why it works there and does not work here:

https://github.com/kk7ds/chirp/blob/master/chirp/drivers/mml_jc8810.py#L470

Reply:
Well, crap. I clearly missed that while I was trying to not re-contaminate with the NEEDS_COMPAT_SERIAL that still existed in my original file.